common/src/main/java/io/druid/metadata/DefaultPasswordProvider.java

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
Hrm, btw, it looks like that wasn't done for other files:

https://github.com/druid-io/druid/blob/master/common/src/main/java/io/druid/common/config/ConfigManager.java

Not sure when it is and isn't used.  We can definitely switch the header around, but I had told @himanshug to use this header because I hadn't noticed that you were using the other one yet...